holmes/plugins/toolsets/prometheus/prometheus.py

```
@@ -752,100 +544,512 @@
+            ):
+                data["_truncated"] = True
+                data["_message"] = (
+                    f"Results truncated at limit={PROMETHEUS_METADATA_API_LIMIT}. Use metric parameter to filter by specific metric name."
+                )
+
+            return StructuredToolResult(
+                status=StructuredToolResultStatus.SUCCESS,
+                data=data,
                 params=params,
             )
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)